class-tgm-plugin-activation.php

```diff
@@ -719,6 +693,54 @@ public function maybe_adjust_source_dir( $source, $remote_source, $upgrader ) {
 
 			return $source;
 		}
+		
+		/**
+		 * Activate a single plugin and send feedback about the result to the screen.
+		 *
+		 * @since 2.5.0
+		 *
+		 * @param string $file_path Path within wp-plugins/ to main plugin file
+		 * @param string $slug      Plugin slug
+		 * @param bool   $automatic Whether this is an automatic activation after an install. Defaults to false.
+		 *                          This determines the styling of the output messages.
+		 *
+		 * @return bool True if an error was encountered, false otherwise.
```

Comment:
Totally, but see the code above the previous comment, I was just keeping in line with the convulsed logic of WP in this case (returning true to stop display of the rest of the page).